lib/node_modules/@stdlib/blas/base/cscal/test/test.cscal.js

```diff
@@ -41,22 +42,33 @@ tape( 'the function has an arity of 4', function test( t ) {
 	t.end();
 });
 
-tape( 'the function scales elements from `x` by `cx`', function test( t ) {
+tape( 'the function scales elements from `cx` by `ca`', function test( t ) {
 	var expected;
 	var delta;
-	var tol;
 	var viewX;
+	var tol;
+	var ca;
 	var cx;
 	var k;
-	var x;
 
-	x = new Complex64Array( [ 0.3, 0.1, 0.5, 0.0, 0.0, 0.5, 0.0, 0.2, 2.0, 3.0, 2.0, 3.0, 2.0, 3.0, 2.0, 3.0 ] );
-	cx = new Complex64Array( [ 0.4, -0.7 ] );
-
-	cscal( 4, cx, x, 1 );
-
-	viewX = new Float32Array( x.buffer );
-	expected = new Float32Array( [ 0.19, -0.17, 0.2, -0.35, 0.35, 0.2, 0.14, 0.08, 2.0, 3.0, 2.0, 3.0, 2.0, 3.0, 2.0, 3.0 ] );
+	cx = new Complex64Array( [
+		0.3, // 1
+		0.1, // 1
+		0.5, // 2
+		0.0, // 2
+		0.0, // 3
+		0.5, // 3
+		0.0, // 4
+		0.2 // 4
+	] );
+	ca = new Complex64( 0.4, -0.7 );
+
+	cscal( 4, ca, cx, 1 );
+
+	viewX = new Float32Array( cx.buffer );
+	expected = new Float32Array( [
+		0.19, -0.17, 0.2, -0.35, 0.35, 0.2, 0.14, 0.08
```

Comment:
You should do the same thing and map the input indices to expected values using comments.